app/views/self_service/advisers/_advisers_table.html.erb

```diff
@@ -22,6 +23,9 @@
               <%= link_to adviser.name, edit_self_service_firm_adviser_path(adviser.firm, adviser), class: 't-edit-link' %>
             </td>
             <td class="t-adviser-postcode" data-dough-filterable><%= adviser.postcode %></td>
+            <td>
+              <%= link_to 'Remove from Directory', self_service_firm_adviser_path(firm, adviser), method: :delete, data: {confirm: 'Are you sure?'}, :class => 'button button--small t-remove-button' %>
```

Comment:
This is very possible using jQuery... http://stackoverflow.com/questions/2153917/how-to-send-a-put-delete-request-in-jquery

All ujs is doing is magically creating a form in the background using Javascript, setting the delete method, and then submitting the form. You can see this in the ujs source here: https://github.com/rails/jquery-ujs/blob/master/src/rails.js#L199

It is simply just not possible to make a 'delete' link without a form. Links do GET requests, and that's it. ujs is just masking this functionality for you, and as a result we're pulling in a 500 line library for no reason.

Seems like you just need a basic onclick listener, see if the result of the `confirm` is truthy and then submit the request using the `delete` method. As a side note, `confirm` is a Javascript feature that has been around since the dark ages, rails just hooks into it.